src/frontend/src/features/layouts/components/mailbox-panel/components/mailbox-labels/components/label-form-modal/index.tsx

```diff
@@ -158,6 +170,22 @@ export const LabelModal = ({ isOpen, onClose, label }: LabelModalProps) => {
                 fullWidth
               />
             </div>
+            <div className="form-field-row">
+               <RhfInput
+                 name="description"
+                 label={t('Description')}
+                 text={form.formState.errors.description?.message && t(form.formState.errors.description.message)}
```

Comment:
IMO I'll do not show this field if is auto labelling feature is not enabled.

Then when the field is in default state, I'll add a `text` message telling the description can be used by AI auto labeling feature as context.